ui/packages/catalog-ui-search/src/main/webapp/component/confirmation/confirmation.less

```diff
@@ -14,20 +14,35 @@
     left: ~'calc(50% - 200px)';
     top: 50%;
     height: auto;
-    width: 400px;
+    width: 500px;
     .changeBackground(@backgroundAccentContent);
     .boxShadow;
   }
 
   .confirmation-prompt {
     width: 100%;
-    text-align: center;
+    text-align: left;
     min-height: @minimumButtonSize;
     padding: 10px;
   }
 
+  .confirmation-header {
+    display: inline-block;
+    font-size: @largeFontSize;
+  }
+
+  .fa-exclamation-triangle {
+    color: orange;
+  }
+
+  hr {
+    margin-top: 10px;
+    margin-bottom: 10px;
+  }
+
   .confirmation-choices {
     white-space: nowrap;
+    padding: 10px;
```

Comment:
✏️ Shouldn't we use less variables for all these hard-coded values so the resize flows more smoothly?

Reply:
`@minimumSpacing` or `@mediumSpacing`